package.json

```diff
@@ -2,18 +2,27 @@
   "name": "hmt-escrow",
   "version": "0.14.5",
   "description": "Launch escrow contracts to the HUMAN network",
-  "main": "truffle.js",
+  "main": "hardhat.config.ts",
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "compile": "truffle compile; truffle migrate",
-    "fix:js": "eslint --fix test/** migrations/**",
-    "lint:js": "eslint test/** migrations/**",
-    "lint": "npm run lint:js",
-    "publish": "truffle publish",
-    "test": "npx truffle test",
-    "coverage": "./node_modules/.bin/solidity-coverage"
+    "compile": "hardhat compile",
+    "deploy": "hardhat run scripts/deploy.ts",
+    "verify": "hardhat verify",
+    "lint": "eslint . --fix",
+    "test": "hardhat test",
```

Comment:
Can we pull the hmt subgraph into the base directory 
so that way we can use the codegeneration at https://github.com/humanprotocol/hmt-escrow/blob/c493a8cc91a23aee34ebe66d23a0ae8749876039/hmt_subgraph/package.json#L5

Reply:
@posix4e I think yes, what do you think if we will leave this tasks to another PR?

Reply:
Yep sounds good!